build-docker.temp-log

```diff
@@ -0,0 +1,1344 @@
+## First Run ##
```

Comment:
Eventually I would remove the log from the PR, it’s pretty big and obfuscates the rest of the changes a bit :)

Reply:
Hi @wdanilo & @piotrMocz, you are welcome guys, glad to help :) I’d say the plan now is to release the first working version of luna in docker with good docs on how to use it, and how to get started with a small app for new users, then we could decide what is the next step from there.

I agree with you guys that that test-app folder shouldn’t be there, when we finish the working docker image, you guys could create a empty repo, and copy the initial code to it, then we add docs to it later on.

For Dockerfile, the norm is to have it in the source code, as it creates an image from the current folder (though not necessary)
Having it in the source code, means that it would be possible to make a small change in the code, and create an image from it without having to publish (for testing purposes)
But I guess you guys already have an existing process for installation, so let’s try to fit it in there.